### ISSUE TITLE
docs on '.dropdown-trigger' unclear

### ISSUE THREAD

**Issue by mkropf** [PolymerElements/iron-dropdown#138]
### Description
the docs refer to `.dropdown-trigger` for relative positioning, however, it does not specify where an element with that class could exist; there is no such reference in the source, are the docs out of date?

**Comment by valdrinkoshi** [comment]
fair point, the README needs to be updated... @mkropf feeling like sending a PR?
`<iron-dropdown>` will position itself according to the `positionTarget`, which defaults to its parent. The other properties for positioning are `vertical/horizontalOffset` and `vertical/horizontalAlign`.
All these properties are inherited from `IronFitBehavior`
